README.md

```diff
@@ -17,4 +17,54 @@ Aside from these lines, all the other code in this repository has been written b
 
 ## Usage
 
-It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
\ No newline at end of file
+It's recommended to use the `blend-settings` UI, instead of the `blend` CLI.
+
+#Installation
+install podman from your distro's repo.
+i.e sudo dnf install podman
```

Comment:
pip install pexpect
cd blend-settings
npm i

Reply:
ah i see

this is useful for gui install :)
but i started with cli, and the cli instructions dont get npm until installing node.js